src/components/avatars/OcAvatarGroup.spec.js

```diff
@@ -41,21 +41,23 @@ describe("OcAvatarGroup", () => {
         users,
         maxDisplayed: 2,
         isTooltipDisplayed: true,
+        accessibleDescription: "List of users",
       },
     })
 
-    expect(wrapper.attributes()["uk-tooltip"]).toMatch("Bob, Marie +3")
+    expect(wrapper.find('.oc-avatar-group').attributes()["uk-tooltip"]).toMatch("Bob, Marie +3")
```

Comment:
We don't have the `uk-tooltip` anymore, needs to be updated

Reply:
```expect(wrapper.attributes()["uk-tooltip"]).toMatch("Bob, Marie +3")```

all those tests should be fine after rebase, already in master